CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Versions
 2020-05-DEV
 -----------
 * Install helm on IBM image
+* Bump PHP minor versions and tools
```

Comment:
Could you please add more details? Will be appreciated on projects update :)

Reply:
should have looked at previous changelogs :facepalm: , it's fixed!